modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectAction/tabs/ActionBuilder.tsx

```diff
@@ -193,43 +201,38 @@ export default function ActionBuilder({
 
 		const {items} = (await response.json()) as {items: ObjectField[]};
 
-		const allFields: ObjectField[] = [];
+		const validFields: ObjectField[] = [];
 
 		items.forEach((field) => {
-			if (
-				field.businessType !== 'Aggregation' &&
-				field.businessType !== 'Relationship' &&
-				!field.system
-			) {
-				allFields.push(field);
+			if (isValidField(field)) {
+				validFields.push(field);
 			}
 		});
```

Comment:
Use  `.filter()` instead:
```suggestion
		const validFields = items.filter(isValidField);
```